CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Breaking Changes
+
+- Structured Logs: Remove `SentrySwiftLog` Integration (#6286)
```

Comment:
Ah my bad sorry. We will copy the full changelog from the alpha release for the next alpha beta. Then we're just going to ditch `Structured Logs: Add `SentrySwiftLog` Integration (#6286)` for this specific release.

I just opened https://github.com/getsentry/sentry-cocoa/pull/6743 to duplicate the changelog. Once we merged the PR, please merge it into this PR, and then remove the `Structured Logs: Add `SentrySwiftLog` Integration (#6286)` from the unreleased section.